library/alloctests/tests/num.rs

```diff
@@ -0,0 +1,69 @@
+use std::fmt::{Debug, Display};
+use std::str::FromStr;
+
+fn assert_nb<Int: ToString + FromStr + Debug + Display + Eq>(value: Int) {
+    let s = value.to_string();
+    let s2 = format!("{}", value);
```

Comment:
Maybe worth doing something like `format!("s: {}.")` to avoid this losing coverage if we ever impl https://github.com/rust-lang/rust/pull/76531, https://github.com/rust-lang/rust/issues/52804, ... -- will make the assertions a bit more complicated, but seems plausibly worth it.

Reply:
Sure, done!